### PR TITLE
patch CVE and sync versioning

### PR DESCRIPTION
**Pull request by YANG-DB**
sync build version number

Signed-off-by: YANGDB <yang.db.dev@gmail.com>

### Description
Patch CVE and sync version declarations 
 
### Issues Resolved
https://github.com/opensearch-project/sql-jdbc/issues/43
 
### Check List
- [x] New functionality includes testing.
  - [x] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -25,7 +25,7 @@ plugins {
 group 'org.opensearch.driver'
 
 // keep version in sync with version in Driver source
-version '2.0.0.0'
+version '1.0.0.0'
```
Comment (penghuo) [inline build.gradle:28]: change to 1.0.0.0?

File: src/main/java/org/opensearch/jdbc/ConnectionImpl.java [hunk 2]
```diff
@@ -507,7 +507,7 @@ public Logger getLog() {
     }
 
     private String getUserAgent() {
-        return String.format("openes-jdbc/%s (Java %s)",
+        return String.format("opensearch-jdbc/%s (Java %s)",
```
Comment (penghuo) [inline src/main/java/org/opensearch/jdbc/ConnectionImpl.java:510]: Are there any clients that rely on string value equality?

File: build.gradle (hunk 1 above)
Comment (joshuali925) [reply build.gradle:28]: latest release is 1.1.0.1, this should be 1.2.0.0?

File: src/main/java/org/opensearch/jdbc/ConnectionImpl.java (hunk 2 above)
Comment (YANG-DB) [reply src/main/java/org/opensearch/jdbc/ConnectionImpl.java:510]: good question  - this originally looks like a type err - should I keep it for the case you mentioned ?

File: build.gradle (hunk 1 above)
Comment (YANG-DB) [reply build.gradle:28]: yes - this actually should be in sync with java/org/opensearch/jdbc/internal/Version.java:line 12 
`  // keep this in sync with the gradle version
    Current(1, 0, 0, 0);
`

File: src/main/java/org/opensearch/jdbc/ConnectionImpl.java (hunk 2 above)
Comment (YANG-DB) [reply src/main/java/org/opensearch/jdbc/ConnectionImpl.java:510]: I'll revert this just to be on the same side

File: build.gradle (hunk 1 above)
Comment (YANG-DB) [reply build.gradle:28]: 1.2.0.0 it is...

**Review by joshuali925 — APPROVED** [review]

**Pull request merged**